mv-openapi.yaml

```diff
@@ -0,0 +1,476 @@
+              $ref: '#/components/links/delete-poll'
+            create-ballot:
+              $ref: '#/components/links/create-ballot'
+            list-poll-results:
+              $ref: '#/components/links/list-poll-results'
+        '400':
+          description: Bad Request. Expect at least 2 proposals and a title.
+  '/polls/{poll-id}':
+    parameters:
+      - in: path
+        name: poll-id
+        required: true
+        schema:
+          type: integer
+          format: int64
+    get:
+      operationId: get-poll
+      summary: Gets a poll by ID
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/poll'
+        '404':
+          $ref: '#/components/responses/poll-not-found'
+    patch:
+      operationId: update-poll
+      summary: Updates a poll
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/poll'
+      security:
+        - {}
+        - pollAuth: []
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/poll'
+    delete:
+      operationId: delete-poll
+      summary: Deletes a poll
+      responses:
+        '200':
+          description: OK
+        '404':
+          $ref: '#/components/responses/poll-not-found'
+      security:
+        - pollAuth: []
+  '/polls/{poll-id}/ballots':
+    parameters:
+      - in: path
+        name: poll-id
+        required: true
+        schema:
+          type: string
+    post:
+      operationId: create-ballot
+      summary: Creates a ballot
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/ballot'
+      security:
+        - {}
+        - participantAuth: []
+      responses:
+        '201':
+          description: Created
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  id:
+                    type: string
+                    description: Id of the created ballot.
+          links:
+            get-ballot:
+              $ref: '#/components/links/get-ballot'
+            delete-ballot:
+              $ref: '#/components/links/delete-ballot'
+  '/polls/{poll-id}/ballots/{ballot-id}':
+    parameters:
+      - in: path
+        name: poll-id
+        required: true
+        schema:
+          type: string
+      - in: path
+        name: ballot-id
+        required: true
+        schema:
+          type: string
+    get:
+      operationId: get-ballot
+      summary: Gets a ballot by ID
+      responses:
+        '200':
+          description: A ballot object
```

Comment:
I think it's way more standard to use the usual http code OK. I've looked at several API. I don't think any tooling script would fail though with a non standard message.

Reply:
On which line this comment belongs to exactly?

Reply:
Good point. It seems that no one (and thus tooling) about it. So let's leave it as you did.